doc/nrf/doc_styleguide.rst

```diff
@@ -202,20 +218,20 @@ File headers and groups
 Functions
 =========
 
-#. Do not use ``@fn``. Instead, document each function where it is defined.
-#. ``@brief`` is mandatory.
+* Do not use ``@fn``. Instead, document each function where it is defined.
+* ``@brief`` is mandatory.
 
-   * Start the brief with the "do sth" form.
+   * Start the brief with the imperative form (for example "do something").
 
      .. code-block:: none
 
         /** @brief Request a read operation to be executed from Secure Firmware.
 
         /** @brief Send Boot Keyboard Input Report.
 
-#. ``@details`` is optional.
-   It can be introduced either by using ``@details`` or by leaving a blank line after ``@brief``.
-#. ``@param`` should be used for every parameter.
+* ``@details`` is optional.
+   Alternatively, you can leave a blank line after ``@brief``.
```

Comment:
```suggestion
   You can introduce the additional information using ``@details`` or by leaving a blank line after ``@brief``.
```
Not happy with the wording. Could maybe be folded more neatly into the previous line?